src/utils.js

```diff
@@ -0,0 +1,35 @@
+import request from 'request-promise';
+import AWS from 'aws-sdk';
+
+import logger from './logger';
+
+const s3 = new AWS.S3();
+
+export const isObjectEmpty = object => object.size === 0;
+
+export const getCompanySlug = object => object.key.split('/')[0];
+
+export const getSrcKey = object => object.key.replace(/\+/g, ' ');
+
+export const getExtension = object => getSrcKey(object).split('.').pop().toLowerCase();
+
+export const doesObjectHaveValidExtension = (object) => {
+	const extension = getExtension(object);
+	const allowedExtensions = ['html', 'htm', 'pdf'];
```

Comment:
We should capture these in a constant.

Reply:
maybe YOU should capture these in a constant.

Reply:
I always do.